src/interfaces/index.ts

```diff
@@ -86,3 +90,136 @@ export type DocumentContextWithSession = DocumentContext & {
 };
 
 export type IncomingMessageWithSession = IncomingMessage & Session;
+
+export interface SessionContents {
+    body: any;
+}
+
+export interface SalesOfferPackage {
+    name: string;
+    description: string;
+    purchaseLocations: string[];
+    paymentMethods: string[];
+    ticketFormats: string[];
+}
+
+export interface Product {
+    productName: string;
+    productPrice: string;
+    productDuration?: string;
+    productValidity?: string;
+}
+
+export interface PeriodProductData {
+    products: ProductDetails[];
+}
+
+export interface FlatFareProductData {
+    products: FlatFareProductDetails[];
+}
```

Comment:
We can remove `PeriodProductData` and `FlatFareProductData` as we didnt end up using these

Reply:
Now removed.